src/app_state.rs

```diff
@@ -0,0 +1,13 @@
+const WEB_SOCKET_URL: &str = "ws://localhost:9091";
```

Comment:
I think, we should add it to the config

Reply:
I agree, it needs to be done in another task